src/smc_sagews/smc_sagews/sage_salvus.py

```diff
@@ -2412,7 +2412,7 @@ def show(*objs, **kwds):
        - svg: (default: True); if True, show 2d plots using svg (otherwise use png)
 
        - d3: (default: True); if True, show graphs (vertices and edges) using an interactive D3 viewer
-           for the many options for this viewer, type 'import graphics; graphics.graph_to_d3_jsonable?'
+           for the many options for this viewer, type 'import smc_sagews.graphics; smc_sagews.graphics.graph_to_d3_jsonable?'
```

Comment:
what I'll be trying to fix here is, that this line-wraps and isn't copy/paste friendly (adds spaces and of course, the line-break ...)